include/zephyr/bluetooth/conn.h

```diff
@@ -2688,6 +2688,28 @@ struct bt_conn *bt_conn_lookup_addr_br(const bt_addr_t *peer);
  */
 const bt_addr_t *bt_conn_get_dst_br(const struct bt_conn *conn);
 
+/** @brief Change the role of the conn.
+ *
+ *  @param conn Connection object.
+ *  @param role The role that want to switch as.
```

Comment:
You should mention what the possible values are. `conn.h` is not an HCI API, so it's not clear if you mean HCI values or something else. And if you mean HCI values you should state where these are defined (`hci_types.h` perhaps?).